docs/tutorials/autogluon-cloud.md

```diff
@@ -1,108 +1,184 @@
+        },
+        {
+            "OutputKey": "RoleARN",
+            "OutputValue": "arn:aws:iam::222222222222:role/ag-cloud-execution-role",
+            "Description": "ARN of the created IAM role for AutoGluon-Cloud to run on SageMaker"
+        }
+    ]
+    ```
+
+:::
+:::{tab-item} CloudFormation (AWS Console)
+:sync: setup-cfn
+1. Download and review the CloudFormation template for the backend of your choice from the [AutoGluon-Cloud repository](https://github.com/autogluon/autogluon-cloud/tree/add-cfn-templates/cloudformation)
+    - Template for [SageMaker](https://raw.githubusercontent.com/autogluon/autogluon-cloud/refs/heads/add-cfn-templates/cloudformation/ag_cloud_sagemaker.yaml)
+    - Template for [Ray (AWS)](https://raw.githubusercontent.com/autogluon/autogluon-cloud/refs/heads/add-cfn-templates/cloudformation/ag_cloud_ray_aws.yaml)
+
+    ```{note}
+    Make sure you review the IAM policy defined in the CloudFormation template, and make necessary changes according to your use case before applying it.
+    ```
+
+2. Log in to the <a href="https://console.aws.amazon.com" target="_blank" rel="noopener noreferrer">AWS Console</a>. Make sure to select the region where you would like to use AutoGluon-Cloud.
+4. Go to <a href="https://console.aws.amazon.com/cloudformation/home#/stacks/create" target="_blank" rel="noopener noreferrer">CloudFormation > Stacks > Create stack</a> and create a stack using the CloudFormation template downloaded in Step 1.
+
+5. After the stack is created, go to the *Outputs* tab and view the **name of the S3 bucket** and the **ARN of the IAM role** created for AutoGluon-Cloud
+    ![img](img/stack-outputs.png)
+
+:::
+:::{tab-item} Manual
+:sync: setup-manual
+1. Create an S3 bucket for AutoGluon-Cloud to store predictors. Replace `S3_BUCKET_NAME` with your preferred name for the bucket.
+    ```bash
+    aws s3 mb s3://S3_BUCKET_NAME
+    ```
+
+2. Generate trust relationship and IAM policy with our utils via the following command
+    ```python
+    from autogluon.cloud import TabularCloudPredictor  # Can be other CloudPredictor as well
+
+    TabularCloudPredictor.generate_default_permission(
+        backend="BACKEND_YOU_WANT",  # We currently support "sagemaker" and "ray_aws"
+        account_id="YOUR_ACCOUNT_ID",  # The AWS account ID you plan to use for CloudPredictor.
+        cloud_output_bucket="S3_BUCKET_NAME"  # S3 bucket name where intermediate artifacts will be uploaded and trained models should be saved. You need to create this bucket beforehand.
+    )
+    ```
+    ```{note}
+    Make sure you review the trust relationship and IAM policy files, and make necessary changes according to your use case before applying them.
+    ```
+    In the following steps, make sure to replace `AUTOGLUON-ROLE-NAME` with your desired role name, `AUTOGLUON-POLICY-NAME` with your desired policy name, and `222222222222` with your AWS account number.
```

Comment:
Preview of the built documentation
![image](https://github.com/user-attachments/assets/ecf74d05-71e2-4d84-914a-fdf461ffb54b)